src/Compatibility/Core/src/iOS/Extensions/FontExtensions.cs

```diff
@@ -1,120 +1,17 @@
-using System.Diagnostics;
-using System.Linq;
-using Microsoft.Maui.Controls.Compatibility.Internals;
-using UIKit;
-using System;
 using Microsoft.Maui.Controls.Internals;
+using UIKit;
 
 namespace Microsoft.Maui.Controls.Compatibility.Platform.iOS
 {
 	public static partial class FontExtensions
```

Comment:
These classes are basically reduce to being extension methods that call into the font manager.